### ISSUE TITLE
Fix test_negative_create_usergroup_{2,3}

### ISSUE THREAD

**Issue by sghai** [SatelliteQE/robottelo#2034]
Stacktrace
```
  File "/usr/lib64/python2.7/unittest/case.py", line 369, in run
    testMethod()
  File "/home/jenkins/shiningpanda/jobs/abb3bf3f/virtualenvs/d41d8cd9/lib/python2.7/site-packages/ddt.py", line 146, in wrapper
    return func(self, *args, **kwargs)
  File "/home/jenkins/workspace/satellite6-automation-downstream-rhel66-ui/tests/foreman/ui/test_usergroup.py", line 77, in test_negative_create_usergroup_2
    make_usergroup(session, org=self.org_name, name=group_name)
  File "/home/jenkins/workspace/satellite6-automation-downstream-rhel66-ui/robottelo/ui/factory.py", line 336, in make_usergroup
    org=org, loc=loc, force_context=force_context)
  File "/home/jenkins/workspace/satellite6-automation-downstream-rhel66-ui/robottelo/ui/factory.py", line 56, in core_factory
    set_context(session, org=org, loc=loc, force_context=force_context)
  File "/home/jenkins/workspace/satellite6-automation-downstream-rhel66-ui/robottelo/ui/factory.py", line 101, in set_context
    session.nav.go_to_select_org(org)
  File "/home/jenkins/workspace/satellite6-automation-downstream-rhel66-ui/robottelo/ui/navigator.py", line 350, in go_to_select_org
    menu_locators['org.select_org'], entity=org
  File "/home/jenkins/workspace/satellite6-automation-downstream-rhel66-ui/robottelo/ui/navigator.py", line 46, in menu_click
    .format(tertiary_menu_locator))
'Tertiary_menu_locator not found: (\'xpath\', "//div[contains(@style,\'static\') or contains(@style,\'fixed\')]//a[@href=\'/organizations/clear\']/../../li/a[contains(.,\'%s\')]")\n-------------------- >>
```

**Comment by sghai** [comment]
Another test is affecting with same issue : test_negative_create_usergroup_3

**Comment by sghai** [comment]
Fixed in PR #2056

**Comment by elyezer** [comment]
#2056 got merged, closing this.
